schedule.md

```diff
@@ -1,3 +1,13 @@
 | Unit |  Date      | Lesson |
 |:----:|:----------:|--------|
 | 0    | 2016-07-18 | [Environment & Tools](lessons/env-and-tools) |
+| 0    | 2016-07-18 | [Intro to Pair Programming]() |
+| 1    | 2016-07-19 | [JS Fundamentals: Workshop Review](lessons/javascript-fundamentals/workshop-review) |
+| 1    | 2016-07-20 | [JS Fundamentals: Types, Strings, and Numbers](lessons/javascript-fundamentals/types-strings-and-numbers) |
+| 1    | 2016-07-21 | [JS Fundamentals: Objects and Arrays](lessons/javascript-fundamentals/objects-and-arrays) |
+| 1    | 2016-07-22 | [Intro To HTML and CSS Part 1](lessons/html-and-css/intro-to-html-and-css-part-1) |
+| 1    | 2016-07-25 | [JS Fundamentals: Functions Deep Dive](lessons/javascript-fundamentals/functions-deep-dive) |
+| 1    | 2016-07-26 | [JS Fundamentals: Advanced Array Methods](lessons/javascript-fundamentals/advanced-array-methods) |
+| 1    | 2016-07-27 | [Intro To HTML and CSS Part 2](lessons/html-and-css/intro-to-html-and-css-part-2) |
+| 1    | 2016-07-28 | [Review/Project]() |
+| 1    | 2016-07-29 | [Asessment]() |
```

Comment:
I personally think "intentionally making mistakes" can be dangerous. I've seen slower students later ask questions making it clear that they interpreted the intentional mistake as correct and it stuck with them, further confusing them / slowing them down. I believe it can be done well, but you have to be very careful to make sure a student that may be tuning in and out (if they are lost/behind/slow) doesn't think even for a second that the mistake is correct.